code/game/objects/items/implants/implant.dm

```diff
@@ -72,7 +72,7 @@
 	return 1
 
 /obj/item/implant/proc/removed(mob/living/source, silent = 0, special = 0)
-	src.loc = null
+	moveToNullspace()

```

Comment:
why does it go into nullspace?

Reply:
It seems like the design is that "bare" implants are not supposed to happen and that they're either inside a dood, in nullspace waiting to be transferred into something else or qdel'd, or in an implant holder.